tox.ini

```diff
@@ -1,24 +1,21 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy
+    py27,py34,py35,py36,pypy,pypy3
```

Comment:
I do not like that the `coverage` environment is missing here.
But thank you for defining the `coverage` environment. This is better than not having it.